source/jsonnet/england-wales/ccs/blocks/individual/ethnic_group.jsonnet

```diff
@@ -9,11 +9,23 @@ local proxyTitle = {
   ],
 };
 
-local englandInstruction = 'Tell the respondent to turn to <strong>Showcard 9E</strong> or show them the answer options below';
-local walesInstruction = 'Tell the respondent to turn to <strong>Showcard 9W</strong> or show them the answer options below';
+local englandInstruction = 'Tell the respondent to turn to <strong>Showcard 9E</strong> or show them the options below';
```

Comment:
I am not sure we should have 6 'if' statements checking for the language, it should only do it once. I reckon we should put all the strings in 2 dicts (Eng and Welsh), then pass the correct dict into the question.  It would also clean up all the local variables. I won't share it on here, but will send my code